lib/mcb/azure.rb

```diff
@@ -88,13 +88,15 @@ def self.configure_for_webapp(webapp:, rgroup: nil, subscription: nil, **_opts)
                                          rgroup: rgroup,
                                          subscription: subscription)
 
-      # TODO: only require confirmation on commands that write to the db
-      print "As a safety measure, please enter the expected RAILS_ENV for #{webapp}: "
-      expected_environment = $stdin.readline.chomp
+      unless %w(qa staging).include?(app_config["RAILS_ENV"])
+        # TODO: only require confirmation on commands that write to the db
```

Comment:
Is the todo still applies?

Reply:
I'd ditch it, I'd say it should go in trello if it needs doing